src/msg/async/ProtocolV1.cc

```diff
@@ -1320,6 +1326,21 @@ void ProtocolV1::reset_recv_state()
   }
 }
 
+void ProtocolV1::log_latency(
+  const char* name,
+  const double& l,
```

Comment:
nit: let's rename to `lat`, `l` is just confusing visually.

Reply:
done